### DISCUSSION TITLE
[Hackathon Use Case] - Capture information from chats to enhance data that is not captured

### DISCUSSION THREAD

**Discussion by maoo** [finos/finos-pmcs#247] [Ideas]
# Capture information from chats to enhance data that is not captured
_From @vquinon (BMO)_

## Business Problem
A lot of missed opportunities (missed tickets/trades) are kept within chats etc, if we start capturing these and tie them to certain securities or sectors, we can use this additional information to derive more accurate hit ratios on clients and securities.

## Expected outcome

- Set up a chat as if it was a BBG chat, use chatGPT to interact with the client and detect when the client wanted to trade, but this didn't complete for various reasons (traded away, price too high etc)
- Store this information as a missed trade with the appropriate attributes, like client, security, reason, date, salesperson, etc
produce visualizations/charts of this additional information
- if you have existing transaction data, this can enhance your KPIs (buy vs sell, done vs missed on a security, done vs missed by a client) and have this in a chart format that updates as new data is streaming in

## Useful apps, systems, projects, technologies, libraries and datasets

- OpenFin/Glue42 (& possibly Finsemble)
- ag-grid
- highcharts / d3 / chartiq / perspective
- ChatGPT

**Comment by robmoffat** [comment]
"Applying ChatGPT to the broker relationship"

**Comment by TheJuanAndOnly99** [comment]
Closing discussion in preparation [for this years hackathon](https://www.finos.org/hosted-events/2024-10-25-citi-india-hackathon-supported-by-finos). Feel free to reopen and update if its still relevant.
